demo/src/scheduleFixedDynamicDispatchTask.js

```diff
@@ -2,75 +2,79 @@ require("@oak-network/api-augment");
 const { rpc, types } = require("@oak-network/types");
 const { ApiPromise, WsProvider, Keyring } = require("@polkadot/api");
 const { waitReady } = require("@polkadot/wasm-crypto");
-const _ = require("lodash");
+const lodash = require("lodash");
```

Comment:
1. Do not change the varaible from `_` to `lodash`. `_` is the norm.
2. Set up this `demo` project with `babel`, the same as xcm-demo, so we can re-use `import`. Let’s keep the code the same format as xcm-demo. All you need are the config file https://github.com/OAK-Foundation/xcm-demo/blob/master/babel.config.js and dependencies in package.json.